### PR TITLE
feat: add data request examples

### PR DESCRIPTION
**Pull request by gabaldon**

### CODE REVIEW

File: src/store/rad.js [hunk 1]
```diff
@@ -468,6 +484,7 @@ export default {
           key: 'templates',
           value: templates,
         })
+        console.log(request)
```
Comment (clbartoli) [inline src/store/rad.js:487]: is this console.log needed?

File: src/constants.js [hunk 2]
```diff
@@ -163,3 +166,21 @@ export const BIRTH_DATE_DELAY_DAYS = 30
...
+    description: 'Bitcoin price in USD',
+    radRequest: bitcoinPrice,
+  },
+  {
+    name: 'Eth price',
+    description: 'Eth price in USD',
+    radRequest: ethPrice,
+  },
+  {
+    name: 'Gold price',
```
Comment (clbartoli) [inline src/constants.js:182]: I would delete the gold example, I think the BTC and ETH are enough

File: src/components/EditorToolBar.vue [hunk 3]
```diff
@@ -156,14 +156,17 @@ export default {
...
-      set() {
-        this.toggleTryDataRequest()
-      },
       get() {
         return this.autoTry
       },
+      set() {
+        this.toggleTryDataRequest()
+      },
     },
```
Comment (Tommytrg) [inline src/components/EditorToolBar.vue:165]: AFAIK we are not changing anything here

File: src/components/steps/Loading.vue [hunk 4]
```diff
@@ -30,6 +31,7 @@ export default {
     sessionId(value) {
       if (value) {
         this.$router.push(`/wallet/transactions?session_id=${this.sessionId}`)
+        this.setDefaultTemplates({ locale: this.locale })
```
Comment (Tommytrg) [inline src/components/steps/Loading.vue:34]: IMO, this logic doesn't belong to the loading component. We can call this method after unlock the wallet in vuex. There we should also check if the wallet already has the default templates before save them.

File: src/store/rad.js [hunk 5]
```diff
@@ -489,6 +505,10 @@ export default {
         session_id: context.rootState.wallet.sessionId,
         key: 'templates',
       })
+      console.log(
+        `getTemplates--${context.rootState.wallet.walletId}-${context.rootState.wallet.sessionId}--`,
+        request,
+      )
```
Comment (Tommytrg) [inline src/store/rad.js:511]:
```suggestion
```

**Review by Tommytrg — CHANGES_REQUESTED** [review]

File: src/components/steps/Loading.vue (hunk 4 above)
Comment (gabaldon) [reply src/components/steps/Loading.vue:34]: #1714

**Review by Tommytrg — APPROVED** [review]

**Comment by Tommytrg** [comment]
Merged in #1715

**Pull request closed without merging**